nixos/modules/rename.nix

```
@@ -71,6 +71,11 @@ with lib;
 
     (mkRemovedOptionModule [ "services" "seeks" ] "")
 
+    (mkRemovedOptionModule [ "security" "hideProcessInformation" ] ''
+        The hidepid module was removed, since the underlying machinery
+        is broken when using cgroups-v2 / systemd.
```

```suggestion
        is broken when using cgroups-v2.
```

cgroupv2 is a kernel feature, not a systemd feature. systemd is only one thing mounting the cgroup hierarchy.

---

changed that